web/src/components/atomic/Icon.vue

```diff
@@ -1,59 +1,59 @@
 <!-- cSpell:ignore radiobox timelapse -->
 <template>
-  <SvgIcon v-if="name === 'duration'" :path="mdiTimelapse" size="20" />
-  <SvgIcon v-else-if="name === 'since'" :path="mdiClockTimeEightOutline" size="20" />
-  <SvgIcon v-else-if="name === 'push'" :path="mdiSourceBranch" size="20" />
-  <SvgIcon v-else-if="name === 'pull-request'" :path="mdiSourcePull" size="20" />
-  <SvgIcon v-else-if="name === 'pull-request-closed'" :path="mdiSourceMerge" size="20" />
-  <SvgIcon v-else-if="name === 'manual-pipeline'" :path="mdiGestureTap" size="20" />
-  <SvgIcon v-else-if="name === 'tag'" :path="mdiTagOutline" size="20" />
-  <SvgIcon v-else-if="name === 'deployment'" :path="mdiPackageVariant" size="20" />
-  <SvgIcon v-else-if="name === 'commit'" :path="mdiSourceCommit" size="20" />
-  <SvgIcon v-else-if="name === 'back'" :path="mdiArrowLeft" size="20" />
+  <SvgIcon v-if="name === 'duration'" :path="mdiTimelapse" size="1.5rem" />
```

Comment:
The default size for icons is still too big. The line height for text with the default font-size is 20px or ~ 1.3rem not 1.5

Reply:
Agreed, well spotted!